docs/cmdline-opts/compressed.d

```diff
@@ -3,5 +3,7 @@ Help: Request compressed response
 Protocols: HTTP
 ---
 Request a compressed response using one of the algorithms curl supports, and
-save the uncompressed document.  If this option is used and the server sends
-an unsupported encoding, curl will report an error.
+save the uncompressed document. Headers are not modified.
```

Comment:
Another suggestion , _save the uncompressed document_ is not always right, the content may not be saved.
```suggestion
automatically decompress the response. Headers are not modified.
```

Reply:
Yeps, that's better!